### DISCUSSION TITLE
Maya 2022 glsl shader

### DISCUSSION THREAD

**Discussion by ypapafi2022** [iimachines/Maya2glTF#178] [Q&A] [answered]
There seems to be an issue while assigning PBR shader/textures on obj. I prepare the OBJ  in substance painter and export to glTF PBR, but in maya it doesn't work.  The steps i follow - mayagltf/assign pbr shader to selection/shader file/

**Comment by ziriax** [comment]
Can you give more details? Error log, OS, output window?

**Reply by ypapafi2022** [reply]
Hi, the message I got is :Not assigning 'u_basecolorTexture' since no texture filename matches pattern 'basecolor/albedo'
probably i did something wrong

**Comment by ziriax** [comment, marked as answer]
Ah that is a know issue. 

Does this help:
https://github.com/iimachines/Maya2glTF/issues/166#issuecomment-913684445
